addons/bestja_offers_moderation/views/offer.xml

```diff
@@ -29,5 +29,18 @@
                 ('state', '=', 'pending'),
             ]</field>
         </record>
+
+        <record id="view_offer_kanban" model="ir.ui.view">
+            <field name="model">offer</field>
+            <field name="inherit_id" ref="bestja_offers.view_offer_kanban"/>
+            <field name="arch" type="xml">
+                <xpath expr="//a[@data-name='set_published']" position="attributes">
+                    <attribute name="groups">bestja_offers_moderation.offers_moderator</attribute>
+                </xpath>
+                <xpath expr="//a[@data-name='set_published']" position="before">
+                    <a t-if="record.state.raw_value == 'unpublished'" data-name="set_pending" data-type="object" class="oe_kanban_action" groups="bestja_organization.coordinators">prześlij do akceptacji</a>
```

Comment:
Change `bestja_organization.coordinators` to `bestja_project.managers` (you don't have to use both since all coordinators have moderator privileges).